test/buildah-bud/apply-podman-deltas

```diff
@@ -140,6 +140,15 @@ errmsg "no such file or directory" \
 skip "N/A under podman" \
      "bud-flags-order-verification"
 
+# TODO
+# All podman tests in CI expects exit code 125, which might not be true since exit code from runtime
+# is relayed as it is without any modification both in `buildah` and `podman`. Following behviour is seen when PR https://github.com/containers/buildah/pull/3809
+# added a test here https://github.com/containers/buildah/blob/main/tests/bud.bats#L3183 which relays exit code from runtime as it
+# is, in case of both `podman` and `buildah`. However apart from this test case no other test case was able to trigger this behviour hence
+# marking this test as an anomaly. Since its debatable if we should override this returned error number or not hence adding a note here.
+skip "podman CI expects all exit code to be 125 this test has anomaly behaviour" \
+     "bud with --add-host"
```

Comment:
@edsantiago Could you please review this line. It seems all of `podman` tests are expected to have return code as `125` in CI however with this new test added in this PR https://github.com/containers/buildah/pull/3809 here https://github.com/containers/buildah/blob/main/tests/bud.bats#L3183 following behavior is changed since error code from test is being relayed directly from runtime so its error code is `1` instead of `125`.

Hence I have marked this as skip in podman since this looks like an anomaly and I'm not sure we should change the error code manually.

Reply:
This comment is misleading: it's not true that "All podman tests in CI expect exit code 125". It would be more accurate to say: "as a general rule, in cases where buildah exits with status 1 or 2, podman exits 125 instead. This is an exception because of (something something runtime)". There's also a typo, should be "behavior".

The rest of your comment is great, the context is very helpful, but please make an effort to restrict text to 80 columns.